index.html

```diff
@@ -206,5 +206,81 @@ <h4>Skills</h4>
         
     </section>
 </section>
+
+<footer>
+        <div class="footspace">
+           <!-- <div class="image01"><img src="assets/Shape.png" alt="#" class="shape00"></div>
+           <div class="image02"><img src="assets/Shape_1.png" alt="#"  class="shape01"></div>
+           --></div> 
+        <div class=" box name"><h2>Get Started</h2></div>
+        <div class=" box intro"><p>Lorem ipsum dolor sit amet,
+             consectetur adipiscing elit, sed do eiusmod tempor
+              incididunt ut labore et dolore magna.</p></div>
```

Comment:
- [ ] [OPTIONAL] Let's make sure `hx` and our `p` tag start with the start of inputs to match Figma design 

Your Implementation | Figma's Design
:--:|:--:
![image](https://user-images.githubusercontent.com/87689547/149041776-6f11569f-980f-4084-bf3a-7a80b947496e.png) | ![image](https://user-images.githubusercontent.com/87689547/149041788-398a5c4e-503a-4276-ad99-dcae693d533f.png)